features/kinderen.feature

```diff
@@ -31,130 +29,122 @@ Functionaliteit: Kinderen van een persoon raadplegen
+      Gegeven de persoon met burgerservicenummer '000000061' heeft een 'kind' met de volgende gegevens
+      | voornamen (02.10) | voorvoegsel (02.30) | geslachtsnaam (02.40) | geboortedatum (03.10) | gemeente document (82.10) | datum document (82.20) | beschrijving document (82.30) | ingangsdatum geldigheid (85.10) | datum van opneming (86.10) |
+      | Daan              | de                  | Vries                 | 20031107              | 0518                      | 20031109               | PL gerelateerde               | 20031107                        | 20031109                   |
+      En het 'kind' is gewijzigd naar de volgende gegevens
+      | gemeente document (82.10) | datum document (82.20) | beschrijving document (82.30) | ingangsdatum geldigheid (85.10) | datum van opneming (86.10) |
+      | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550002                       |
+      | burgerservicenummer | 000000061                       |
       | fields              | kinderen                        |
-      Dan bevat het antwoord GEEN kinderen
+      Dan heeft de response een persoon zonder 'kind' gegevens
 
-    @gba
     Scenario: met fields vragen om een gegeven zonder waarde
-      Gegeven de persoon met burgerservicenummer 999996150 heeft de volgende kinderen in de registratie
-      | Categorie | Burgerservicenummer (01.20) | Voornamen (02.10) | Voorvoegsel (02.30) | Geslachtsnaam (02.40) | Geboortedatum (03.10) | Gemeente document (82.10) | Datum document (82.20) | Beschrijving document (82.30) | Ingangsdatum geldigheid (85.10) | Datum van opneming (86.10) |
-      | 9         |                             | Bert              |                     | Jansen                | 20110422              | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
+      Gegeven de persoon met burgerservicenummer '000000073' heeft een 'kind' met de volgende gegevens
+      | voornamen (02.10) | geslachtsnaam (02.40) |
+      | Bert              | Jansen                |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 999996150                       |
+      | burgerservicenummer | 000000073                       |
       | fields              | kinderen.burgerservicenummer    |
-      Dan heeft de persoon met burgerservicenummer '999996150' een 'kind' zonder gegevens
+      Dan heeft de response een persoon zonder 'kind' gegevens
```

Comment:
moet nog worden aangepast conform #1198: Dan heeft de response een persoon met een 'kind' zonder gegevens.

Heeft veel overlap met Scenario: kind volledig onbekend, veld met onbekend waarde wordt niet gevraagd met fields

Reply:
Is nu geworden:

`Dan heeft de response een persoon zonder 'kind' gegevens`